x-pack/plugin/ml/src/main/java/org/elasticsearch/xpack/ml/MachineLearning.java

```diff
@@ -842,4 +848,16 @@ static long machineMemoryFromStats(OsStats stats) {
         }
         return mem;
     }
+
+    @Override
+    public List<NamedWriteableRegistry.Entry> getNamedWriteables() {
```

Comment:
We have a data frame plugin but this is part of ml?

Reply:
The data frame plugin is for data frame _transforms_, and is basic licensed.  The data frame _analytics_ is in the ML plugin and is platinum licensed.

But the fact that you asked this shows that the docs that explain the distinction to users will need careful thought.